README.md

```diff
@@ -11,36 +11,71 @@ This repo is split into two parts:
 - Ansible playbooks for integrating with OpenStack services, and creating 
   software middleware platforms on top of ALaSKA infrastructure.
 
+Installation
+------------
+
+Inside a Python 2.7+ virtual environment,
```

Comment:
Isn't this creating the virtualenv?

Reply:
The role? Don't think so... how would it?

Reply:
No, the commands that are being run create the virtualenv.

Reply:
Ah, is this point about making the note clearer? i.e. Create a virtual environment by.....